hack/packages/package-tools/templates/package.go

```diff
@@ -1,14 +1,37 @@
-#@ load("@ytt:data", "data")
+package templates
+
+const (
```

Comment:
optional: You could use `go:embed` instead of constants. More idiomatic and it can still be part of a separate go module.

Reply:
done